src/components/GenericDropdown.tsx

```diff
@@ -10,8 +10,15 @@ interface Props {
 
 const DropdownButton = styled.button`
 	display: flex;
+	margin-left: auto;
```

Comment:
Not sure if styling is within the scope of this ticket, but the height of the dropdown button was poking a little bit past the bottom of the header ;-;
```suggestion
	margin-left: auto;
	height: 100%;
```

Reply:
I suggest adding this as well